examples/dynamic-config-fs/verify.sh

```diff
@@ -19,7 +19,7 @@ curl -s http://localhost:19000/config_dump \
     | grep '"address": "service1"'
 
 run_log "Set upstream to service2"
-sed -i s/service1/service2/ configs/cds.yaml
+docker exec -t dynamic-config-fs_proxy_1 sed -i s/service1/service2/ /var/lib/envoy/cds.yaml
```

Comment:
s/-t/-T/

Reply:
could you use `docker-compose` - i think its simpler/more reliable for this

```
docker-compose exec -T proxy ...
```

Reply:
sure :)